code/__DEFINES/traits.dm

```diff
@@ -214,7 +214,14 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 /// Prevents plasmamen from self-igniting if only their helmet is missing
 #define TRAIT_NOSELFIGNITION_HEAD_ONLY "no_selfignition_head_only"
 #define TRAIT_NOGUNS "no_guns"
+///This carbon doesn't get hungry, has no stomach or appendix
 #define TRAIT_NOHUNGER "no_hunger"
+///This carbon doesn't bleed, has no heart
+#define TRAIT_NOBLOOD "noblood"
+///This carbon doesn't have an appendix. Which is funny, because that doesn't change a whole lot out of dodging appendicitis
+#define TRAIT_NOAPPENDIX "no_appendix"
+///This carbon doesn't have a tongue.
+#define TRAIT_NOTONGUE "no_tongue"
```

Comment:
these shouldnt be mob traits, bring em back to species traits, even if theyre stupid